starters/svelte-kit-scss/src/routes/(examples)/api-example/+page.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
   import Greeting from '$lib/components/Greeting/Greeting.svelte';
-
+  
```

Comment:
you can just `git checkout src/routes/(examples)/api-example/+page.svelte` to get rid of this file change